compose/service.py

```diff
@@ -479,6 +488,7 @@ def build(self, no_cache=False):
             stream=True,
             rm=True,
             nocache=no_cache,
+            fileobj=self._get_fileobj(),
```

Comment:
Do you think I should update this to use dockerfile (even know it's not released yet?)